src/components/detailsPage/pages/resources/ResourcesMainContent.tsx

```diff
@@ -0,0 +1,96 @@
+import Title from 'components/Title/Title';
+import Divider from 'components/divider/Divider';
+import Subtitle from 'components/subtitle/Subtitle';
+import DataField from 'data-field/DataField';
+
+import GMarkdown from '../../../../common/markdown/GMarkdown';
+import { ISelfDescription } from '../../utils/dataMapper';
+
+import styles from './ResourcesMainContent.module.css';
+
+export interface IDetailsCard extends ISelfDescription {
+}
+
+interface IDetailsCardProps {
+    cardData: IDetailsCard;
+}
+
+export default function ResourcesMainContent({ cardData }: Readonly<IDetailsCardProps>) {
```

Comment:
The name should be in singular `ResourceMainContent`. You just show the content of a single resource.